tests/apps/test_operators.py

```diff
@@ -0,0 +1,335 @@
+import numpy as np
+import unittest
+import testutils
+import tinychain as tc
+import torch
+from torch.autograd import grad
+from tinychain.util import form_of, hex_id
+
+TENSOR_URI = str(tc.uri(tc.tensor.Dense))
+HOST = tc.host.Host('http://127.0.0.1:8702')
+ENDPOINT = '/transact/hypothetical'
+
+
+class OperatorTests(unittest.TestCase):
```

Comment:
I think this should actually go in `tests/client` since it's a test of functionality implemented in the Python client and doesn't need to load a cluster configuration